explore-analyze/visualize/charts/line-charts.md

```diff
@@ -0,0 +1,233 @@
+---
+navigation_title: Line charts
+applies_to:
+  stack: ga
+  serverless: ga
+description: Instructions and best practices for building line charts with Kibana Lens in Elastic.
+---
+
+# Build line charts with {{kib}}
+
+Line charts are ideal for visualizing how metrics evolve over time, spotting seasonal patterns, and detecting spikes or regressions at a glance. Use them for KPIs like response time, error rate, throughput, or utilization, and compare multiple series or previous periods on the same chart.
+
+You can create line charts from any numeric data using aggregations (for example, `Average`, `Percentile`, `Counter rate`) or with custom [formulas](../lens.md#lens-formulas). Line charts usually show time on the x-axis, but they can also display data grouped by numbers or categories.
```

Comment:
>Line charts usually show time on the x-axis, but they can also display data grouped by numbers or categories.

In general is not great to have categories on the X axis of a line chart. The only "acceptable" categories are the ones that represent a continuous or sequential progression like month of a year, or quarters (usually always related to time).

If the connection between categories is not meaningful like "small," "medium," and "large", is incorrect to suggest a continuous, linear relationship and a steady rate of change between them.

Also if the spacing between categories is not equal or relevant can be misleading.